updatefinish.lua

```diff
@@ -0,0 +1,144 @@
+--Alter the date which the updating region process finishes at.
```

Comment:
Am I correct in assuming that this requires you to run it when the screen is advancing (so you only have a short window when you can run it, and have to wait for the world to load first)? If so, it might be nice to allow it to be run *before* the world update screen appears, and intercept that screen when it does show up. A state change handler would probably work for that.

Also, this is another example of documentation in different places - the first line here won't show up on https://dfhack.readthedocs.io/.

Reply:
Not sure exactly how to make it do that but I'll look into it, outside of super quick updating worlds it gives you plenty of time.